src/modelconfig.cpp

```diff
@@ -406,6 +406,24 @@ Status ModelConfig::parseNode(const rapidjson::Value& v) {
         modelVersionPolicy = ModelVersionPolicy::getDefaultVersionPolicy();
     }
 
+    SPDLOG_DEBUG("Specified model parameters:");
+    SPDLOG_DEBUG("model_basepath: {}", getBasePath());
+    SPDLOG_DEBUG("model_name: {}", getName());
+    SPDLOG_DEBUG("batch_size: {}", getBatchSize());
+    SPDLOG_DEBUG("shape:");
+    for (auto& shapeMap : getShapes()) {
+        std::stringstream shapeStream;
+        std::copy(shapeMap.second.shape.begin(), shapeMap.second.shape.end(), std::ostream_iterator<size_t>(shapeStream, " "));
```

Comment:
Shape have two modes: `fixed` and `auto`. We should only print shape if mode is fixed. If auto, we can print "auto".
```
if (shapesMap.second.shapeMode == Mode::AUTO) {
// print "auto"
} else {
// print shape
}
```